.github/workflows/copy-images-to-dockerhub.yaml

```diff
@@ -11,23 +11,27 @@ permissions:
   id-token: write #required for GCP Workload Identity federation
 
 jobs:
-  copy-images:
-    strategy:
-      fail-fast: false
-      matrix:
-        IMAGE_NAME: [validator, forge, tools, faucet, indexer, node-checker]
-
+  wait-for-images-to-have-been-build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-
       - name: Wait for images to have been built
-        timeout-minutes: 20
+        timeout-minutes: 30
         uses: lewagon/wait-on-check-action@v1.0.0
         with:
           ref: ${{ github.ref }}
           check-regexp: "rust-images.*"
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 30 # wait 30 seconds between making polling API calls, default is 10 but we ran in the past into rate-limiting issues with too frequent polling
```

Comment:
Why not make it 60 seconds? This doesnt seem super time sensitive

Reply:
hmm idk. I think 30 seconds is more reasonable. I agree that it's not suuuper time sensitive usually but when you're in the middle of a SEV it can be a bit surprising/annoying when you have to wait 1 minute for the image to appear on dockerhub vs just in ECR. With 30 seconds that delay fades in the background.